openpype/hosts/nuke/plugins/create/create_write_still.py

```diff
@@ -1,12 +1,10 @@
-from collections import OrderedDict
-
 import nuke
 
-from openpype.hosts.nuke.api import plugin
 from openpype.hosts.nuke.api.lib import create_write_node
+from openpype.hosts.nuke.plugins.create import create_write_render
```

Comment:
I don't think we should import from other plugins at all, it adds import dependency between plugins so you (or others) may not know that changes in `CreateWriteRender` will affect other plugins. If you want extract common logic then it should be inside `openpype.hosts.nuke.api.<something>`.